applications/Interstellar-network2.md

```diff
@@ -0,0 +1,201 @@
+# Interstellar - Wallet Phase 2
+
+
+- **Team Name:** Interstellar
+- **Payment Address:** TBD
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
```

Comment:
Thanks for changing the scope! Can you update the application to reflect a level 2 grant? Also, can you update the payment address now or do you plan to submit an amendment at a later time?

Reply:
And can you change the checked box on the PR as well? Thanks!